### PR TITLE
Update dev installation instructions since spyder 4 is default now

### LINKED ISSUE #102: Update Development Version Installation Instructions For Spyder 4 As Default
## Issue Description

Because Spyder 4 is the default version in the repository, some instructions that distinct version 3 and 4 are no longer necessary.

### Where is the relevant portion of the documentation located?

In doc/installation.rst there is a long note about how to install development environment for spyder 4:

```rst
#. Install the Spyder `requirements`_.

   The recommended and easiest way to do this is with ``conda`` (although experts may prefer ``pip``). In a fresh environment (``conda create -n your-name-here -c conda-forge python=3``, then ``activate`` it), run the following:

   .. code-block:: bash

      conda install -c conda-forge/label/beta spyder=4.0.0b1
      conda install -c conda-forge python-language-server
      conda remove spyder

   This installs all of Spyder's dependencies into the environment along with the stable/packaged version of Spyder, and then removes Spyder itself.

   .. note::

      Following the separation in v3.3 of Spyder's console code into its own package, ``spyder-kernels``, you'll need to have the corresponding version of it available—``0.x`` for Spyder 3 (``3.x`` branch), and ``1.x`` for Spyder 4 (``master`` branch).
      The above procedure will install the ``0.x`` version; to test the ``master`` branch (Spyder 4), you'll need to install the corresponding ``1.x`` version of ``spyder-kernels``.
      This can be done via two methods: installing the ``1.x`` version via ``conda``:

      .. code-block:: bash

         conda install -c spyder-ide spyder-kernels=1.*

      or ``pip``:

      .. code-block:: bash

         pip install spyder-kernels==1.*

      (and using the same respective command, replacing ``1`` with ``0``, to switch back to the Spyder 3 version), or by ``clone``-ing the `spyder-kernels git repository`_ to somewhere on your path and checking out the appropriate branch (``0.x`` or ``master``) corresponding to the version of Spyder (3 or 4) you would like to run, and running the commend ``pip install -e`` at the root.
      For any non-trivial development work, keeping two separate virtual environments (with ``conda-env`` or ``venv``) for Spyder 3 and 4 makes this process much quicker and less tedious.
```

### What would you suggest it be replaced with?

Delete these notes and fix the commands to proper ones.

### Other information

* Is the problem specific to documentation for a particular version of Spyder?

* Are you planning on submitting a Github pull request with a suitable change?
Yes.

* Anything else you'd like to add?

### PR DESCRIPTION
**Pull request by qrqiuren**
### Pull Request Checklist

* [X] Read and followed this repo's [Contributing Guidelines](https://github.com/spyder-ide/spyder/blob/master/CONTRIBUTING.md)
* [X] Based your PR on the latest version of the correct branch (master or 3.x)
* [X] Checked your writing carefully for correct English spelling, grammar, etc
* [X] Described your changes and the motivation for them below
* [x] Noted what issue(s) this pull request resolves, creating one if needed

## Description of Changes

Because Spyder 4 is the default version in the repository, some instructions that distinct version 3 and 4 are no longer necessary. In this PR, some installation instructions for development version has been modified. See the issue below.

### Issue(s) Resolved

Fixes #102

### CODE REVIEW

File: doc/installation.rst [hunk 1]
```diff
@@ -252,35 +252,17 @@ In summary:
 
 #. Install the Spyder `requirements`_.
 
-   The recommended and easiest way to do this is with ``conda`` (although experts may prefer ``pip``). In a fresh environment (``conda create -n your-name-here -c conda-forge python=3``, then ``activate`` it), run the following:
+   The recommended and easiest way to do this is with ``conda`` (although experts may prefer ``pip``). To install Spyder in a fresh environment, run the following (omitting the first two commands if you'd like to use an existing one):
```
Comment (CAM-Gerlach) [inline doc/installation.rst]:
```suggestion
   The recommended and easiest way to do this is with ``conda`` (although experts may prefer ``pip``).
   To install Spyder in a fresh environment, run the following (omitting the first two commands if you'd like to use an existing one):
```

I believe this error was present in the original, but per the docs style guide sentences should each be on their own line.

**Comment by goanpeca** [comment]
@CAM-Gerlach would you like to review the PR ?

**Comment by CAM-Gerlach** [comment]
@goanpeca Ummmm ^^ :rofl:

**Comment by CAM-Gerlach** [comment]
Given this has gone 6 months without a response and the proper updates are already underway, I'll close this now. Sorry about that.

**Pull request closed without merging**